docs/source/en/api/pipelines/ldm3d_diffusion.mdx

```diff
@@ -0,0 +1,59 @@
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# LDM3D
+
+LDM3D was proposed in [LDM3D: Latent Diffusion Model for 3D](https://arxiv.org/abs/2305.10853) by Gabriela Ben Melech Stan, Diana Wofk, Scottie Fox, Alex Redden, Will Saxton, Jean Yu, Estelle Aflalo, Shao-Yen Tseng, Fabio Nonato, Matthias Muller, Vasudev Lal
+The abstract of the paper is the following:
+
+*This research paper proposes a Latent Diffusion Model for 3D (LDM3D) that generates both image and depth map data from a given text prompt, allowing users to generate RGBD images from text prompts. The LDM3D model is fine-tuned on a dataset of tuples containing an RGB image, depth map and caption, and validated through extensive experiments. We also develop an application called DepthFusion, which uses the generated RGB images and depth maps to create immersive and interactive 360-degree-view experiences using TouchDesigner. This technology has the potential to transform a wide range of industries, from entertainment and gaming to architecture and design. Overall, this paper presents a significant contribution to the field of generative AI and computer vision, and showcases the potential of LDM3D and DepthFusion to revolutionize content creation and digital experiences. A short video summarizing the approach can be found at [this url](https://t.ly/tdi2) *
+
+
+*Overview*:
+
+| Pipeline | Tasks | Colab | Demo
+|---|---|:---:|:---:|
+| [pipeline_stable_diffusion_ldm3d.py](https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_ldm3d.py) | *Text-to-Image Generation* | - | -
+| [pipeline_stable_diffusion_ldm3d_img2img.py](https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_ldm3dn_img2img.py) | *Image-to-Image Text-Guided Generation* | - |-
+
+## Tips
+
+- LDM3D diffusion is conceptually exactly the same as [Stable Diffusion](./stable_diffusion/overview).
+
+- *Run LDM3D*
+
+LDM3D can be tested very easily with the [`StableDiffusionLDM3DPipeline`]
+
+- *How to load and use different schedulers.*
+
+The LDM3D diffusion pipeline uses [`DDIMScheduler`] scheduler by default.
+To use a different scheduler, you can either change it via the [`ConfigMixin.from_config`] method or pass the `scheduler` argument to the `from_pretrained` method of the pipeline. For example, to use the [`EulerDiscreteScheduler`], you can do the following:
+
+```python
+>>> from diffusers import StableDiffusionLDM3DPipeline
```

Comment:
On a second look I saw that the pipeline requires manual approval from the pipeline authors. I think we need to clarify that here.

Reply:
Hi @sayakpaul -- I made a request on huggingface.co for access to Intel/ldm3d -- I'd love to try this out and wandering if I could get approved. Thanks!

Reply:
Not in my hands, unfortunately :(

Reply:
I enabled the access automatically 
Could you please confirm that you can load the model properly?

Reply:
Fantastic. In the mode card, I think it might make sense to also include some results to make it for the users to understand them a little bit better. Also would be nice to include the known biases and limitations in the model card itself as I already see there's a TODO for that.

Reply:
Hmm, I just opened it from an incognito window and it seems like the gating has been removed so, it should work right out of the box. 

Thanks for making changes to the model card. I'd additionally suggest to add the results after the code snippet (the generated image and the depth map).